itests/kit/node_unmanaged.go

```diff
@@ -938,20 +1012,43 @@ func (tm *TestUnmanagedMiner) generateProveCommit(
 
 	tm.t.Logf("Miner %s: Running proof type %d SealCommitPhase2 for sector %d...", tm.ActorAddr, proofType, sectorNumber)
 
-	sectorProof, err := ffi.SealCommitPhase2(scp1, sectorNumber, actorId)
-	req.NoError(err)
+	var sectorProof []byte
+
+	if proofType.IsNonInteractive() {
+		circuitProofs, err := ffi.SealCommitPhase2CircuitProofs(scp1, sectorNumber)
```

Comment:
So we have the ProveCommit for NI-PoRep here. But what do we replace the `PreCommit` phase with ?

Reply:
See above, absolutely nothing, it just flows through PC1, PC2, C1, C2(w/ raw circuit proofs output)